package-managers/nuget.md

```diff
@@ -89,24 +89,43 @@ To work with public feeds, we just leave the credentials empty and Rider will co
 
 ### NuGet CLI
 
-#### Public feed
+#### Private feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
-#### Private feed
+#### Public feed
 
 Details on how to find the feed URL can be found in the [Find the feed URL](#find-the-feed-url) section.
 
 ### dotnet CLI
 
-#### Public feed
+In this section we are going to add our packages via the dotnet CLI. We assume that the dotnet CLI tools are installed. If this is not the case, our friends at Microsoft can help us with that via [this link](https://www.microsoft.com/net/core).

```

Comment:
`via` -> `using`
`If this is not the case, our friends at Microsoft can help us with that via [this link](https://www.microsoft.com/net/core).` -> `Make sure to [install the .NET Core tooling](https://www.microsoft.com/net/core).` or something like that (SEO-friendly, and people with a screen reader will also have something meaningful instead of `this link`)